maintainers/maintainer-list.nix

```diff
@@ -1,63 +1,64 @@
-/* List of NixOS maintainers.
```

Comment:
Please do not format the `maintainer-list.nix` file, because this might cause merge issue with everyone editing it!

Reply:
Sorry, that was careless of me! Force pushed without formatting